docs/build/README.md

```diff
@@ -7,6 +7,10 @@ description: Documentation to start building applications on Filecoin.
 
 Filecoin is for the builders. If you are excited about the potential of leveraging the Filecoin protocol and decentralized storage network to build game-changing applications, you've come to the right place.
 
+:::tip
+Use one of the community built tools and services to get started. [Take a look now →](../store/readme.md) 
+:::
+
 ## Get started
```

Comment:
@johnnymatthews I think this is an area where we should mention web3.storage

Reply:
@pooja - @johnnymatthews wrote it that way in the first iteration and then got pushback on prioritizing it rather than linking to a full list including Estuary and community-built solutions. This is the crux of the current debate.

Reply:
What I meant was I think it should be listed as a bullet point in `## Get started`. Looks like it didn't show up here properly. But it's fine for now, we just need _something_ on the docs site that mentions web3.storage!